vignettes/MsBackendMetaboLights.Rmd

```diff
@@ -0,0 +1,186 @@
+---
+title: "Seamless Integration of Mass Spectrometry Data from MetaboLights"
```

Comment:
I do think the backend helps to reach a "seamless integration" but by itself not really. 
Together with re-building the full MsExperiment() with sample data  then starting the preprocessing. That is seamless integration i think no ?  And that is really really cool. 
Kind of continuing on this, does it make sense to separate this backend in one package and then the MsExperiment() object creation in another ?.. I don't know what to think. I just find it a bit convoluted... wouldn't it make sense to regroup them somehow ? Just raising the idea.

Reply:
or 'importing', 'loading',..? but yes otherwise that fits better i think. 
No no i agree completely with you. But i'll think about if there is a possible better option. It's just  a lot of packages overall and i wonder if we are not complexifying the ecosystem, but it is the best option for now. We can discuss it further later on :)

Reply:
I know, we had discussions on whether more or less packages would be better. I'm on the more than less packages side - so only those users using one certain type of data (like MetaboLights) need to install/load a specific package. But yes, we should/could discuss this further. 

The *MsIO* package is currently more a playground to develop the import/export functions independently (without affecting e.g. the *Spectra* or *MsExperiment* packages) - in future we might to move the functionality into the respective packages... but still unsure about that.

Reply:
yes maybe once we have it (MsIO) more developed we can try to "map" out how everything could be distributed essentially. I do like the idea of the import/export in one package though, it's also much easier to explain to people